samples/joke/README.md

```diff
@@ -14,44 +14,179 @@ cluster. You can just follow the steps below to get started quickly:
 
 - Connect to your cluster of choice using `kubectl/oc`, select the appropriate namespace/project.
   The operator will automatically connect to that cluster/namespace combination when started.
-- Run `mvn install` on the parent directory to build the project locally.
-- Run `mvn package` to build the operator. This will automatically generate several resources for
+- Run `mvn install` on the parent directory to build the project locally. This will automatically generate several resources for
   you, in particular the CRDs associated with the custom resources we will be dealing with. These
-  CRDs are generated in `target/classes/META-INF/fabric8` and come in two flavors: `v1`
-  and `v1beta1` versions which correspond to the versions of the CRD spec. We recommend you use
+  CRDs are generated in `target/manifests/` and come in `v1` version which correspond to the versions of the CRD spec. We recommend you use
```

Comment:
Is there no way to generate everything in only one directory, e.g. `target/kubernetes`? Or does it make it more difficult to process the files needed to generate the bundle?

Reply:
If we generate everything in only one directory, we could not easily select the files that will be bundled. Also, the `manifests` folder is a convention of OLM. 
Finally, I would like to improve this to not generate the folder `manifests`, but to generate `bundle/manifests`, `bundle/metadata` and `bundle/bundle.Dockerfile`, so we won't need this step any longer: https://github.com/quarkiverse/quarkus-operator-sdk/pull/238/files#diff-7482aed8473830546fbd8ac45b5cc158781d08fcd8682410d7dd32246f84a881R77